src/client/components/Editor.tsx

```diff
@@ -32,6 +34,37 @@ class Editor extends React.PureComponent<EditorProps> {
         }
     }
 
+  componentDidMount(): void {
+      // TODO: why does using gutterclick yield weird highlighting?
+      this.aceComponent.current.editor.on("gutterclick", function(e) {
+        var target = e.domEvent.target;
+
+        if (target.className.indexOf("ace_gutter-cell") == -1){
+            return;
+        }
+
+        if (!e.editor.isFocused()){
+            return;
+        }
+
+        if (e.clientX > 25 + target.getBoundingClientRect().left){
+            return;
+        }
```

Comment:
We should also figure out what this check is trying to do, and write it in terms of widths instead of using magic numbers